eeps/eep-0009.md

```diff
@@ -140,32 +140,36 @@ the same as for string with the following exceptions:
 3. The new functions substitute and globally_substitute should be
    added.
 
-       substitute(OldBinary, Key, Replacement)-> NewBinary
+   3.1. `substitute/3`
+
+        substitute(OldBinary, Key, Replacement)-> NewBinary
+
+            OldBinary, NewBinary, Replacement = binary()
+            Keys = binary() | [ binary() ] | MatchSpec
+            MatchSpec = tuple() as returned by binary:match_compile/1
 
-       OldBinary, NewBinary, Replacement = binary()
-       Keys = binary() | [ binary() ] | MatchSpec
-       MatchSpec = tuple() as returned by binary:match_compile/1
+      Creates a binary NewBinary from OldBinary by substituting the
+      first occurrence of any of the binaries in Keys in OldBinary
+      with the Replacement binary.
 
-   Creates a binary NewBinary from OldBinary by substituting the
-   first occurrence of any of the binaries in Keys in OldBinary
-   with the Replacement binary.
+      The Replacement binary need not have the same size as the matched
+      Key.
 
-   The Replacement binary need not have the same size as the matched
-   Key.
+      Example:
 
-   Example:
+          > binary_string:substitute(<<"cat anf dog">>,<<"anf">>,<<"and">>).
+          [<<"cat and dog">>]
 
-        > binary_string:substitute(<<"cat anf dog">>,<<"anf">>,<<"and">>).
-        [<<"cat and dog">>]
+   3.2. `globally_substitute/3`
```

Comment:
This was the "hack" I found to separate this code block from the previous; otherwise we have to break some other rules. Doesn't seem to bad, I think.